### ISSUE TITLE
Nvim-bqf doesn't work with fzf anymore

### ISSUE THREAD

**Issue by ovidiu** [kevinhwang91/nvim-bqf#149]
### Neovim version (nvim -v | head -n1)

NVIM v0.10.0

### Operating system/version

Ubuntu 22.04

### How to reproduce the issue

1. Have  `junegunn/fzf` commit `83b6033` or newer
2. Open quickfix
3. Press zf

### Expected behavior

Fzf prompt shows up and I can filter the list

### Actual behavior

Nvim seems to freeze for a second, then gets back to normal. No fzf prompt though.

**Comment by kevinhwang91** [comment]
can't reproduce, provide your `FZF_DEFAULT_OPTS` value.

**Comment by ovidiu** [comment]
I don't have `FZF_DEFAULT_OPTS` set, but I figured it out.

The issue was between `junegunn/fzf` (as an nvim plugin) and the `fzf` binary I had installed on my system.

Since commit `83b6033`  `junegunn/fzf` adds a `--no-tmux` flag when calling `fzf`, but the version I had installed (`0.52.1`) did not support it and threw an error. My neovim setup included `folke/noice.nvim` which swallowed up the error (it was there in `:messages`, I just didn't think to look). Anyway, I upgraded the `fzf` binary to `0.53.0` and the issue went away.

Sorry about the false alarm.
